<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Myvec</class>
 <widget class="QDialog" name="Myvec">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>向量计算</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="InitButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>定义向量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LookButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>135</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查看向量值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>向量加法</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SubButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>265</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>向量减法</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DestroyButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>销毁向量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>395</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>清空向量</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InitEdit2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>请输入变量值，例如(1,2,3)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ResultEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>525</y>
     <width>470</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddEdit2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>参与运算变量</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SubEdit2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>265</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>参与运算变量 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SubEdit3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>265</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>存储结果变量</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddEdit3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>200</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>存储结果变量</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CosEdit1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>参与运算变量 </string>
   </property>
  </widget>
  <widget class="QPushButton" name="CosButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>向量夹角</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CosEdit2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>参与运算变量 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LookEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>135</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>请输入变量名 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SubEdit1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>265</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>参与运算变量 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DestroyEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>460</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>请输入变量名 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddEdit1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>参与运算变量</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ClearEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>395</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>请输入变量名 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InitEdit1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入变量名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>525</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>结果</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
